spec/rubocop/cop/style/hash_syntax_spec.rb

```diff
@@ -891,6 +891,21 @@
         RUBY
       end
 
+      it 'does not register an offense when one line condition follows' do
+        expect_no_offenses(<<~RUBY)
+          foo value: value if boo
```

Comment:
Good catch! This is trivia, `bar` is more common than `boo`.

```suggestion
          foo value: value if bar
```

There is the RFC 3092 developed in April Fool's Day :-)
https://datatracker.ietf.org/doc/html/rfc3092

Reply:
I did not know about the RFC 😅😀.